coffee/chosen.proto.coffee

```diff
@@ -392,8 +392,9 @@ class @Chosen extends AbstractChosen
 
   single_deselect_control_build: ->
     return unless @allow_single_deselect
-    @selected_item.down("span").insert { after: "<abbr class=\"search-choice-close\"></abbr>" } unless @selected_item.down("abbr")
-    @selected_item.addClassName("chosen-single-with-deselect")
+    unless @selected_item.down('.search-choice-close')
+      @selected_item.down('span').insert { after: '<a class="search-choice-close"></a>' }
```

Comment:
Lines 28-30 of `sass/chosen.scss` force a cursor to all links within `.chosen-container`, but I agree that no href is less semantic.